src/kili/gateways/kili_api_gateway/issue/__init__.py

```diff
@@ -26,7 +26,7 @@ def create_issues(
     ) -> List[Issue]:
         """Send a GraphQL request calling createIssues resolver."""
         created_issue_entities: List[Issue] = []
-        with tqdm.tqdm(total=len(issues)) as pbar:
+        with tqdm.tqdm(total=len(issues), desc="Creating issues") as pbar:
```

Comment:
I also added the message on issue creation, it gives a nicer tqdm bar with the description. Very user-friendly, we should use it from now on.
<img width="1039" alt="Capture d’écran 2023-08-29 à 10 52 54" src="https://github.com/kili-technology/kili-python-sdk/assets/82399630/abf67da7-c6ed-4aa4-a23a-b009d6acec91">

Reply:
I agree.